src/Api/Controllers/OrganizationConnectionsController.cs

```diff
@@ -200,10 +197,29 @@ private async Task ValidateBillingSyncConfig(OrganizationConnectionRequestModel<
         }
 
         var data = typedModel.ToData(organizationConnectionId);
+        var billingSyncKey = data.Config as BillingSyncConfig;
```

Comment:
Is is possible to just pass in the `ValidateBillingSyncConfig` function as the argument for `validateAction` here? That's what we are doing for the creation of the Connection and I think it would work for updating them too. https://github.com/bitwarden/server/blob/01d1756c5656f266de4771ca9610c0e7b9dfaf34/src/Api/Controllers/OrganizationConnectionsController.cs#L70